app/views/users/show.html.erb

```diff
@@ -57,6 +57,20 @@
       <% end %>
       </div>
 
+      <% if @user.user_websites.size > 0 %>
+      <div>
+	<p><strong>External links</strong></p>
+	<table class="table is-with-hover">
+	  <% @user.user_websites.first(3).each do |w| %>
+	  <tr>
+	    <td><%= w.label %></td>
+	    <td><%= link_to w.url, w.url, rel: 'nofollow' %></td>
+	  </tr>
+	  <% end %>
+	</table>
```

Comment:
Some weird indentation here - looks like you might be using tab stops instead of spaces for indentation.